packages/ia-user-list/.github/workflows/ci.yml

```diff
@@ -0,0 +1,26 @@
+name: App CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 16.x
+
+    - name: Install dependencies
+      run: yarn install
+
+    - name: Run tests
+      run: yarn run test
+
+    - name: Upload reports
+      run: npx codecov
```

Comment:
I'd like for all of these GitHub Actions to actually work if we were to check it in. if not, let's remove them